### DISCUSSION TITLE
test-notifications not working?

### DISCUSSION THREAD

**Discussion by aminoplacid** [juftin/camply#326] [General]
Returning to this project after a while away.  Was running under a hosted python platform.  I installed the latest version (0.32 I think). I am trying to test notifications with comply test-notifications --notifications telegram and am met with this:

06:22 ~ $ camply test-notifications --notifications telegram
2024-03-08 06:27:41,810 [  CAMPLY]: camply, the campsite finder ⛺️
usage: camply [-h] [--version] {campsites,recreation-areas,campgrounds,configure} ...
camply: error: argument command: invalid choice: 'test-notifications' (choose from 'campsites', 'recreation-areas', 'campgrounds', 'configure')
2024-03-08 06:27:41,816 [  CAMPLY]: Exiting camply 👋

Any ideas?

**Comment by juftin** [comment]
Can you double check the `--version` on this? My first guess is that you've got an outdated version of camply - there seems to be a number of CLI options missing. 

Maybe you have Python < 3.8 so you're not getting the latest camply when you pip install?
